moveit_core/robot_state/test/robot_state_test.cpp

```diff
@@ -614,6 +614,21 @@ std::size_t generateTestTraj(std::vector<std::shared_ptr<robot_state::RobotState
   return traj.size();
 }
 
+TEST_F(OneRobot, testGenerateTrajectory)
```

Comment:
You really like it the verbose way! A unittest for a helper function for unittests...

Reply:
Test all the things! :-P